RabinKarpAlgorithm/RabinKarp.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RabinKarpAlgorithm
+{
+    public class RabinKarp
+    {
+        int mod = 113;
+        public int GetPatternCount(string baseString, string pattern)
+        {
+            int baseStringLen = baseString.Length;
+            int patternLen = pattern.Length;
+
+            char[] possibleMatchChar = new char[patternLen];
+
+            int patternHash = HashMethod(pattern);
+
+            int cnt = 0;
```

Comment:
cnt za sta ?

Reply:
Broj match-ova, u sustini to je povratna vrednost, inkrementuje se svaki put kada se pattern nadje u baseString-u. Preimenovacu ga u nesto tipa matchCount, mislim da bi to bilo jasnije.